includes/transformer/class-base.php

```diff
@@ -236,6 +236,18 @@ public function to_activity( $type ) {
 	protected function get_locale() {
 		$lang = \strtolower( \strtok( \get_locale(), '_-' ) );
 
+		$lang = apply_filters_deprecated(
```

Comment:
I think we would have to test if the item is a post before. I think not every item has an ID